coursedata/adventures/de.yaml

```diff
@@ -603,164 +609,195 @@ adventures:
+                    Das ist etwas langweilig!
+                    In Level 2 kann sie ihre Nase in alle Richtungen strecken.
 
-                    Use 90 to turn a quarter. We call this degrees. A full turn is 360 degrees.
-                    Can you make a figure with this code? Maybe a triangle or a circle?
+                    Nutze 90, um wieder eine viertel Drehung zu machen.
+                    Diese Angabe ist in "Grad (°)".
+                    Eine volle Drehung um sich selbst sind 360° (360 Grad).
+                    Kannst Du mit diesem Code eine bestimmte Form zeichnen?
+                    Ein Dreieck zum Beispiel, oder einen Kreis?
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    print Drawing figures
-                    angle is 90
-                    turn angle
+                    print Zeichne Formen
+                    Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
                 start_code: |
-                    print Turtle race!
-                    hoek is 90
-                    turn hoek
+                    print Schildkrötenrennen!
+                    Winkel is 90
+                    turn Winkel
                     forward 25
             3:
                 story_text: |
-                    ## Searching turtle
-                    In this level you can use use `random` with the drawing turtle. A random choice makes the turtle walk a different path each time.
-                    Use `at random` to choose a value from a list. You can copy and paste lines 2 and 3 to create a longer random path.
+                    ## Suchende Schildkröte
+                    In diesem Level kannst Du den Befehl `random` nutzen, während die Schildkröte etwas zeichnet.
+                    Durch eine zufällige Wahl sieht die Zeichnung jedes Mal anders aus.
+                    Nutze `at random` um einen zufälligen Wert einer Liste zu wählen.
+                    Wenn Du die Zeilen 2 und 3 kopierst und wieder einfügst, kannst Du einen längeren Pfad erzeugen.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    angles is 10, 50, 90, 150, 250
-                    turn angles at random
+                    Winkel is 10, 50, 90, 150, 250
+                    turn Winkel at random
                     forward 25
                     ```
 
                 start_code: |
-                    angles is 10, 50, 90, 150, 250
-                    turn angles at random
+                    Winkel is 10, 50, 90, 150, 250
+                    turn Winkel at random
                     forward 25
```

Comment:
It's unfortunate that "Winkel" is the same in plural and singular, so the `angle` vs. `angles` distinction cannot be made so easily in German. Should we maybe deviate from standard German grammar and make it `Winkels`, as in Haskell and some German dialects / some variants of colloquial German? Or should we use a more complex variable name like `WinkelAuswahl`?